### ISSUE TITLE
Crash while trying to select other branch of production

### ISSUE THREAD

**Issue by tractortractor** [ElmarWueest/Recexplo#16]
```
  58.777 Error MainLoop.cpp:1035: Exception at tick 1113: Error while running event RecExplo::on_gui_click (ID 1)
__RecExplo__/lua/recipe_cal.lua:401: attempt to perform arithmetic on local 'ingredient_amount' (a nil value)
```

How to reproduce:
1. Disable all mods.
2. Install/enable "Recipe Explorer".
3. Start new game and open mod gui.
4. Select Stack filter inserter.
5. Start recording.
6. Select Electronic circuit in recipe.
7. Select Copper cable in recipe.
8. In "Selected item" section select Stack filter inserter again.
9. Select Stack inserter in recipe.

**Comment by ElmarWueest** [comment]
I reproduced the bug and will fix it in the next few days.

**Comment by ElmarWueest** [comment]
will be fixed in the new version 0.16.11

**Comment by ElmarWueest** [comment]
Thank you for your bug reports.
